Lang/Spanish.pj.Lang

```diff
@@ -419,7 +420,7 @@
 //Edit Cheat
 #1027#  "Editar Truco"
 #1028#  "Actualizar Truco"
-#1029#  "El Truco ha sido cambiado.\n\n¿Deseas actualizar?"
+#1029#  "El Truco ha sido cambiado.\n\n¿Quieres actualizar?"

```

Comment:
@MELERIX We had discussed this topic before.
"Deseas" is more appropriate for an option offering in a software application cation menu.

Please take any professional application translated to Spanish, and the definition for "desear" in the RAE dictionary as reference.

"Desear" does not semantically translate as "to desire" in this context.